### ISSUE TITLE
1.6新版本的bug

### ISSUE THREAD

**Issue by WangYneos** [codehz/mcpeserver#45]
上次的玄学问题重装系统解决了
这次来报告一下bug
1，部分人很难进去(算我一个)(特别是死了一次以后)
但是大部分用Windows10版的没问题
2，部分模组会导致崩溃(已发现白名单)
3，在某些特殊情况下，服务器不会开放端口(删地图重新上传就好了)(motd都看不到)
(用的腾讯云200mbps的网络，应该不是网络问题)

**Comment by WangYneos** [comment]
进不去是两种情况
1卡在定位服务器/生成世界
2无法连接到世界

**Comment by codehz** [comment]
@WangYneos 一直都有的问题，但是我修复不好，所以 wont fix

**Comment by keke1229488344** [comment]
@WangYneos 是的，证实一下，我也有这个问题（我还以为就我一个人有这个问题。。），这个问题确实存在。
你说的两种情况我安卓和UWP版本都存在。第一种情况一般要退出重进好几次能解决，第二种情况我这边情况是wifi和流量进不去，目前只有电信卡流量可以进在少部分情况进去，不知道与网络有没有关系，总之就是很难进去服务器，真的很迷。
希望有解决办法~

**Comment by keke1229488344** [comment]
@WangYneos 服务器不会开放端口那个你检查下权限问题，我一般出现这个问题就是刚换了地图权限没有设置好。`chown mcpeserver:mcpeserver /srv/mcpeserver -R`

**Comment by WangYneos** [comment]
应该不是权限问题，上传了地图我都设定了权限的，但是一些地图不知原因会导致服务器不把端口开放，内网也进不了

**Comment by WangYneos** [comment]
另外，服务器创建的地图放到手机上会显示由更新版本游戏保存了地图，无法加载

**Comment by database64128** [comment]
那么请分享会引起“问题”的地图，让大家试试看

Sent from my iPhone

On Sep 1, 2018, at 08:54, WangYneos <notifications@github.com<mailto:notifications@github.com>> wrote:


应该不是权限问题，上传了地图我都设定了权限的，但是一些地图不知原因会导致服务器不把端口开放，内网也进不了

**Comment by WangYneos** [comment]
就手机上创建的地图偶尔会出现

**Comment by WangYneos** [comment]
你放一下试试

**Comment by WangYneos** [comment]
比如这个地图就无法被手机加载
survival_20180901090234.zip
https://pan.baidu.com/s/1LEXTU26rAMbcfX76Q3glsg 密码：2333

**Comment by database64128** [comment]
我用Win10版试了下，没有出现任何问题。手机版的话，我不怎么想安装iTunes，所以等其他人尝试的结果吧。按理说，手机端和Win10端都是一套代码编译的，不应该有行为差异啊。

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

________________________________

**Comment by WangYneos** [comment]
你可以正常加载这个地图吗

**Comment by WangYneos** [comment]
对了，你们有没有遇到死了不能复活，退出不能重进

**Comment by xiaoxi654** [comment]
@WangYneos 我记得之前有看到过

**Comment by database64128** [comment]
@WangYneos 是的，就是你的地图，Win10版正常加载，退出可以重进。但是用/kill @s 自杀后，就会循环死亡，点击重生仍然死亡，关掉重进就进不去了。

**Comment by database64128** [comment]
我觉得是bugjump日常特性，只有等修复了。

**Comment by WangYneos** [comment]
你放到服务器上试试，我找朋友也是没问题，服务器死了就加不进去了

**Comment by WangYneos** [comment]
手动导入地图无法链接

[root@XSBA mcpeserver]# sudo -u mcpeserver mcpeserver run -profile bedwars
I [core] Loading profile: bedwars
N [core] Loading Bridge...
I [core] Bridge Version: 0.1.6
I [DBUS] Starting...
N [core] Loading Bedrock Server...
N [ModLoader] Loading mod: /srv/mcpeserver/user/mods/mods_dragon_fight-1.so

N [ModLoader] Loading mod: /srv/mcpeserver/user/mods/libplayer_support.so

N [ModLoader] Loading mod: /srv/mcpeserver/user/mods/mods_uuid.so
E [ModLoader] Failed to load mod /srv/mcpeserver/user/mods/mods_uuid.so: Cannot
load library: reloc_library[1373]:     0 cannot locate '_ZNK6Entity10getNameTagE
v'...

N [ModLoader] Loading mod: /srv/mcpeserver/user/mods/libform_support.so

N [ModLoader] Loading mod: /srv/mcpeserver/user/mods/mods_teleport-1.so
E [ModLoader] Failed to load mod /srv/mcpeserver/user/mods/mods_teleport-1.so: C
annot load library: reloc_library[1373]:     0 cannot locate '_ZN6Entity8getLeve
lEv'...

N [ModLoader] Loading mod: /srv/mcpeserver/user/mods/mods_chat_log.so

N [ModLoader] Loading mod: /srv/mcpeserver/user/mods/mods_tellraw.so

N [ModLoader] Loading mod: /srv/mcpeserver/user/mods/mods_spam_detector.so

N [ModLoader] Loading mod: /srv/mcpeserver/user/mods/mods_welcome.so
E [ModLoader] Failed to load mod /srv/mcpeserver/user/mods/mods_welcome.so: Cann
ot load library: reloc_library[1373]:     0 cannot locate '_ZNK6Entity10getNameT
agEv'...

N [ModLoader] Loading mod: /srv/mcpeserver/user/mods/mods_blacklist.so
E [ModLoader] Failed to load mod /srv/mcpeserver/user/mods/mods_blacklist.so: Ca
nnot load library: reloc_library[1373]:     0 cannot locate '_ZN20ServerNetworkH
andler13_onPlayerLeftEP12ServerPlayer'...

I [ModLoader] Loaded 10 Mod
D [Bridge] Minecraft is at offset 0xf321b000
I [Bridge] Game version: 1.6.0
I [Bridge] Applying patches
T [AppPlatform] Vtable size = 241
T [AppPlatform] createDeviceID:
T [Bridge] Loading server properties
T [AppPlatform] getCurrentStoragePath: /srv/mcpeserver/data
T [Bridge] Initializing resource loaders
T [AppPlatform] getSystemRegion: 0xdeadbeef
T [Bridge] Initializing ResourcePackManager
T [Bridge] Initializing PackManifestFactory
T [Bridge] Initializing SkinPackKeyProvider
T [Bridge] Initializing PackSourceFactory
T [Bridge] Initializing ResourcePackRepository
T [AppPlatform] getInternalStoragePath: /srv/mcpeserver/data
T [AppPlatform] getInternalStoragePath: /srv/mcpeserver/data
T [AppPlatform] getInternalStoragePath: /srv/mcpeserver/data
T [AppPlatform] getInternalStoragePath: /srv/mcpeserver/data
T [AppPlatform] getInternalStoragePath: /srv/mcpeserver/data
T [AppPlatform] getInternalStoragePath: /srv/mcpeserver/data
T [AppPlatform] getInternalStoragePath: /srv/mcpeserver/data
T [Bridge] Adding vanilla resource pack
T [Bridge] Adding world resource packs
D [Bridge] Initializing SaveTransactionManager
D [Bridge] Initializing ExternalFileLevelStorageSource
D [Bridge] Initializing ServerInstance
T [Bridge] Loading language data
I [Bridge] Server initialized
mcpe://mcpeserver@XSBA$

服务器链接不上(motd看不见)

另外一个地图正常(服务器创建)

**Comment by codehz** [comment]
@WangYneos 先确定是不是权限的问题

**Comment by WangYneos** [comment]
不是，是偶然性的，偶尔突然一次

**Comment by Unknown4k** [comment]
btw is the software Multi threaded? Like can it use all available cores?

**Comment by codehz** [comment]
@Unknown4k Yes! you can use up to 24 core (tested in my local test machine)

**Comment by Unknown4k** [comment]
I'm glad to hear that! :+1:

**Comment by keke1229488344** [comment]
我是motd看得见但是连接不上....内网不行，外网只有流量可以进。。。
至于地图死一次不能连，是退出重进卡定位吗？如果是，目前我只能找到一个临时的解决办法（也不算办法的办法），就是编辑地图然后清除玩家数据（参考我之前提的issue）。

**Comment by WangYneos** [comment]
是的

**Comment by WangYneos** [comment]
谢谢@keke1229488344
